tools/jlm-hls/jlm-hls.cpp

```diff
@@ -102,6 +102,7 @@ main(int argc, char ** argv)
     }
 
     jlm::hls::VerilatorHarnessHLS vhls;
+    vhls.SetIncludeFileName(commandLineOptions.OutputFiles_.base());
```

Comment:
Remove the `SetIncludeFileName()` method and hand the file name over in the constructor (which you have to add to `VerilatorHarnessHLS`).

Reply:
Changed